From 7e6abc85011baedd81ceda824921357ac175f57e Mon Sep 17 00:00:00 2001
From: Craig Small <csmall@enc.com.au>
Date: Mon, 19 May 2014 22:56:50 +1000
Subject: [PATCH 07/21] Fixed v4 versus v6 problem

fuser -n tcp 80 always worked because the v4 or v6 problem didnt hit
single ports, but any ipv4 or ipv6 remote address wouldn't work.
Caught by Coverity CID46269
---
 src/fuser.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fuser.c b/src/fuser.c
index 8d9bd52..4e16d67 100644
--- a/src/fuser.c
+++ b/src/fuser.c
@@ -528,7 +528,7 @@ parse_mounts(struct names *this_name, struct device_list **dev_list,
 
 #ifdef WITH_IPV6
 int
-parse_inet(struct names *this_name, const int ipv6_only, const int ipv4_only,
+parse_inet(struct names *this_name, const int ipv4_only, const int ipv6_only,
 	   struct ip_connections **ip_list, struct ip6_connections **ip6_list)
 #else
 int parse_inet(struct names *this_name, struct ip_connections **ip_list)
-- 
2.5.2

